modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/router/AndesMessageRouter.java

```diff
@@ -144,7 +144,7 @@ public void addMapping(String bindingKey, StorageQueue queue) throws AndesExcept
      * @throws AndesException when there are active subscriptions
      */
     public void removeMapping(String bindingKey, StorageQueue queue) throws AndesException {
-        if (queue.getBoundedSubscriptions().isEmpty()) {
+        if (queue.getBoundSubscriptions().isEmpty()) {
```

Comment:
Can 'queue.getBoundSubscriptions()' be actually empty when this method is called?